### DISCUSSION TITLE
MakeFile not being produced

### DISCUSSION THREAD

**Discussion by hudkinsnoah** [JSBSim-Team/jsbsim#1043] [Q&A] [answered]
I am trying to update the newest version of JSBSim but I am not able to make the project with CMake (which is desired over visual studio). I follow the instructions of making a build directory, changing directories into the new build folder, running ``cmake ..`` and then once that finishes, I run ``make``. Whenever I run ``make``, I get the message ``make: *** No targets specified and no makefile found. Stop.``. Is there something that I am doing incorrectly when trying to build? I am trying to build version 1.2.0 and am using windows 10. 

I have seen that a possible reason for this error is because a make file is not being produced. I am not sure if that is the case here though

**Comment by bcoconni** [comment, marked as answer]
Hi @hudkinsnoah,

Even though you are not using Visual Studio, I assume you are using the Microsoft C++ compiler. In such a case the command to build JSBSim is
```pwsh
cmake --build . --config RelWithDebInfo
```
This command should be run from the `build` folder and will build all the libraries and executables under the folder `build\RelWithDebInfo`. Of course, if you want to get a debug (resp. release) build, you should replace `RelWithDebInfo` with `Debug` (resp. `Release`)

**Reply by hudkinsnoah** [reply]
Thank you! I just wanted to double check if this is targeting 32 or 64 bit? Is there a place where all of the command arguments are listed that I could reference in order to find some more that might be helpful?

**Reply by hudkinsnoah** [reply]
I actually was able to figure this out, thanks again!
